addons/web_tour/static/src/tour_service/tour_interactive.js

```diff
@@ -102,30 +116,40 @@ export class TourInteractive {
     }
 
     setActionListeners() {
-        if (this.anchorEl) {
-            this.anchorEl = this.getAnchorEl(this.anchorEl, this.currentAction.event);
-            const consumeEvents = this.getConsumeEventType(this.anchorEl, this.currentAction.event);
-            this.removeListeners = this.setupListeners({
-                anchorEl: this.anchorEl,
-                consumeEvents,
-                onMouseEnter: () => this.pointer.showContent(true),
-                onMouseLeave: () => this.pointer.showContent(false),
-                onScroll: () => this.updatePointer(),
-                onConsume: () => {
+        this.anchorEls.forEach((anchorEl, index) => {
```

Comment:
Separate the behaviour for the first element and the setup of the listeners. Lot of index === 0